rllib/algorithms/slateq/slateq_torch_policy.py

```diff
@@ -391,7 +391,7 @@ def postprocess_fn_add_next_actions_for_sarsa(
 ) -> SampleBatch:
     """Add next_actions to SampleBatch for SARSA training"""
     if policy.config["slateq_strategy"] == "SARSA":
-        if not batch["dones"][-1] and policy._no_tracing is False:
+        if not batch.is_terminated_or_truncated() and policy._no_tracing is False:
```

Comment:
This is a new API for SampleBatch to make these queries more unified and easier for the user.